sys/kern/kgprof.c

```diff
@@ -2,11 +2,12 @@
 #include <sys/context.h>
 #include <sys/gmon.h>
 #include <sys/kmem.h>
+#include <sys/timer.h>
 #include <sys/thread.h>
 #include <machine/vm_param.h>
 
-static gmonparam_t _gmonparam = {.state = GMON_PROF_NOT_INIT};
-static gmonhdr_t _gmonhdr = {.profrate = CLK_TCK};
+gmonparam_t _gmonparam = {.state = GMON_PROF_NOT_INIT};
```

Comment:
If you made it global please add `extern` definition in header.

Reply:
Already done.